infra/gcp/terraform/k8s-infra-prow-build/provider.tf

```diff
@@ -23,18 +23,18 @@ This file defines:
 terraform {
 
   backend "gcs" {
-    bucket = "k8s-infra-tf-prow-clusters"
-    prefix = "k8s-infra-prow-build/prow-build" // $project_name/$cluster_name
+    bucket = "k8s-infra-terraform"
```

Comment:
This is a state bucket in the k8s-infra-seed project instead of the kubernetes-public